kafka-connect-s3/src/test/resources/log4j.properties

```diff
@@ -21,3 +21,5 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
 log4j.logger.org.apache.kafka=ERROR
+#log4j.logger.io.confluent.connect.s3=DEBUG
+#log4j.logger.io.findify.s3mock=DEBUG
```

Comment:
we can probably remove those lines from the artifacts that get shipped.

Reply:
These are properties are in the test resources. We don't ship those IIRC